### PR TITLE
Replace ilm/slm with their full names

### PR DESCRIPTION
**Pull request by andreidan**
This also fixes some doc typos "reponse" -> "response".

### CODE REVIEW

File: docs/reference/tab-widgets/troubleshooting/data/start-ilm.asciidoc [hunk 1]
```diff
@@ -61,14 +61,14 @@ The reponse will look like this:
 // end::cloud[]
 
 // tag::self-managed[]
-<<ilm-start, Start ILM>>:
+<<ilm-start, Start Index Lifecycle Management>>:
```
Comment (gmarouli) [inline docs/reference/tab-widgets/troubleshooting/data/start-ilm.asciidoc:64]: I am curious why are we specifically typing here `Index Lifecycle Management` and not `{ilm}` like we do in the `start-slm` doc?

**Comment by andreidan** [comment]
@elasticmachine update branch

File: docs/reference/tab-widgets/troubleshooting/data/start-ilm.asciidoc (hunk 1 above)
Comment (gmarouli) [reply docs/reference/tab-widgets/troubleshooting/data/start-ilm.asciidoc:64]: I see, that's why in the slm it is `<<ilm-start, Start>> {slm}`. I would suggest to do the same here so we are consistently using the macros.

**Review by gmarouli — APPROVED** [review]
LGTM, just a small suggestion about using a macro.

File: docs/reference/tab-widgets/troubleshooting/data/start-ilm.asciidoc (hunk 1 above)
Comment (andreidan) [inline docs/reference/tab-widgets/troubleshooting/data/start-ilm.asciidoc:64]:
```suggestion
<<ilm-start, Start>> {ilm}:
```

**Pull request merged**